lib/fluent/supervisor.rb

```diff
@@ -1196,6 +1205,28 @@ def build_system_config(conf)
       system_config
     end
 
+    def build_additional_configuration
+      if @system_config.config_include_dir&.empty?
+        $log.info :supervisor, 'default configuration include directory was disabled'
+        return
+      end
+      begin
+        Find.find(@system_config.config_include_dir) do |path|
+          next if File.directory?(path)
+          next unless [".conf", ".yaml", ".yml"].include?(File.extname(path))
+          # NOTE: both types of normal config (.conf) and YAML will be loaded.
+          # Thus, it does not care whether @config_path is .conf or .yml.
+          $log.info :supervisor, 'loading additional configuration file', path: path
```

Comment:
> $log.info :supervisor, 'loading additional configuration file', path: path

I have noticed that these logs are not logged in the case of `reload_config`.

It is because `:supervisor` limits the message to be logged only when the process is a supervisor.

Since `reload_config` is not recommended, this may be acceptable as a specification.
I shared it anyway.